kong/plugins/cors/handler.lua

```diff
@@ -137,7 +137,9 @@ function CorsHandler:access(conf)
     set_header("Access-Control-Max-Age", tostring(conf.max_age))
   end
 
-  return kong.response.exit(NO_CONTENT)
+  set_header("Content-Length", "0")
```

Comment:
`kong.response.exit` without a body will already set this header appropriately, we can remove this.

Reply:
applied with https://github.com/Kong/kong/pull/4029/commits/58949d2d24c38ea79a588a43327102d3a2ed8624